requirements.txt

```diff
@@ -1,4 +1,6 @@
 # build dependencies
+pip>=22.3 # Due to a bug in `pip install -e .` vs read-only system-wide site-packages
```

Comment:
Actually, this is not a bad idea since editable install bug is only for users installing from source,  and the recommended procedure involves `pip install requirements.txt` first 👍

Reply:
We really should sort these sources of requirements. I would suggest moving as much as we can into `setup.py` since we can not rely on .toml entirely because of the version parsing that we do..

Reply:
Sure, but as you realized `setup.py` is limited.

Reply:
We can keep requirements.txt for buildsystem updates and setup.py for everything else.